cmake/cfe-common.doxyfile.in

```diff
@@ -2,12 +2,7 @@
 # Project related configuration options, shared for all cFE doxygen outputs
 #---------------------------------------------------------------------------
 @INCLUDE_PATH          = @MISSION_SOURCE_DIR@
-DOXYFILE_ENCODING      = UTF-8 
```

Comment:
Were all deleted settings the defaults?

Reply:
Yes

Reply:
Where can a user go find what the defaults are?

Reply:
doxygen will create a config file with the defaults, `doxygen -g configfile`

Reply:
Would it be worthwhile to add a comment that says "to see defaults run doxygen -g configfile" ?

As we experienced last week, doxygen knowledge is sparse, and this might help.

Reply:
It's on the man page...